utils/frame/benchmarking-cli/src/pallet/mod.rs

```diff
@@ -166,4 +166,9 @@ pub struct PalletCmd {
 	/// template for that purpose.
 	#[clap(long)]
 	pub no_storage_info: bool,
+
+	/// A path to a `.json` file with existing benchmark results. When specified the benchmarks
+	/// are not actually executed, and the data for the analysis is read from this file.
+	#[clap(long)]
+	pub override_results: Option<PathBuf>,
```

Comment:
About the naming: `--from-json` or `--json-input`? Otherwise it sounds like you would just change the final output and not use that as input.